packages/web/src/components/FetchConfigProvider.tsx

```diff
@@ -25,6 +25,29 @@ export const FetchConfigProvider: React.FunctionComponent<{
   renderLoading = () => null,
   ...rest
 }) => {
+  const fetchConfigValue: FetchConfig = {
+    uri: `${window.__REDWOOD__API_PROXY_PATH}/graphql`,
+  }
+
+  if (typeof useAuth === 'undefined') {
+    return <FetchConfigContext.Provider value={fetchConfigValue} {...rest} />
+  }
+
+  return (
+    <FetchConfigProviderWithAuth
+      useAuth={useAuth}
+      renderLoading={renderLoading}
+      fetchConfigValue={fetchConfigValue}
+      {...rest}
+    />
+  )
+}
+
+const FetchConfigProviderWithAuth: React.FunctionComponent<{
+  useAuth: () => AuthContextInterface
+  renderLoading: () => React.ReactElement | null
+  fetchConfigValue: FetchConfig
+}> = ({ useAuth, renderLoading, fetchConfigValue, ...rest }) => {
   const { loading, isAuthenticated, getToken, type } = useAuth()
```

Comment:
Can't we just default these to `undefined` if `useAuth?.()` is undefined? And then keep the rest of the code exactly like it was?

Reply:
Absolutely